plasmapy/simulation/abstractions.py

```diff
@@ -54,3 +54,142 @@ class AbstractTimeDependentSimulation(AbstractSimulation):
     """
 
     pass
+
+
+class AbstractNormalizations(ABC):
+    """
+    An abstract base class to represent the normalizations of systems of
+    equations describing plasmas.
+
+    Warnings
+    --------
+    This interface is unstable and is subject to change.
+    """
+
+    @property
+    @abstractmethod
+    def current_density(self) -> u.A / u.m ** 2:
+        """The current density normalization."""
+        pass
```

Comment:
Just a friendly reminder that we decided to go with Python's standard of using `...` with abstract methods. :)

```suggestion
    @property
    @abstractmethod
    def current_density(self) -> u.A / u.m ** 2:
        """The current density normalization."""
        ...
```

Reply:
And if I remember correctly, it was me who suggested that, wasn't it...  :grimacing:

Reply:
It was.  I tried to find the old comment thread, but no such luck. 🤣